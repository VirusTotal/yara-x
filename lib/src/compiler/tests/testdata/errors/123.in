rule test {
  strings:
    $a = /{(foo|bar)/
  condition:
  	$a
}