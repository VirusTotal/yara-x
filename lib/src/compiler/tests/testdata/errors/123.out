error: invalid regular expression
 --> line:3:11
  |
3 |     $a = /{(foo|bar)/
  |           ^ repetition operator missing expression
  |
  = note: did you mean `\{` instead of `{`?